_methods/participatory-design.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: card
+title: Participatory design
+permalink: /fundamentals/participatory-design/
+redirect_from:
+  - /participatory-design/
+description: Design processes that invite participation are more sustainable in the long run.
+category: Fundamentals
+what: An approach to design emphasizing shared ownership, education, and collaboration.
+why: To ensure more just, equitable, and sustainable design processes. To encourage the creation of products and services reflecting the preferences of the people who must use them.
+timeRequired: N/A
+---
+
+## How to do it
+
+<ol>
+  <li>Identify what the group hopes to accomplish.</li>
+  <li>Identify the “core team,” the group of people who will directly participate in the design process. Unless the initiative is small, members of the core team will need to act as representatives for larger groups of stakeholders; for example, designers will often be asked to speak on behalf of users and their needs. Note: The core team will likely change over time!</li>
```

Comment:
So, for example, in classic participatory design designers would invite representative users into the process instead of speaking _for_ them.

Reply:
We don't normally do this, though when we do I've seen it done with a `-public` channel in Slack (which I really like).

Reply:
Which to me means that it's not really participatory design, and so we should rename this to avoid confusing people who are familiar with "real" participatory design.

Reply:
How do designers make "speaking for others" participatory? Does that contradict the aims of collaboration?